index.bs

```diff
@@ -346,6 +346,11 @@ A <dfn interface>DatagramDuplexStream</dfn> is a generic duplex stream.
 interface DatagramDuplexStream {
   readonly attribute ReadableStream readable;
   readonly attribute WritableStream writable;
+
+  undefined setIncomingExpirationDuration(double ms);
+  undefined setOutgoingExpirationDuration(double ms);
```

Comment:
These look like simple setters, and should therefore be read-write attributes instead of methods, I claim. E.g. using latest naming from https://github.com/w3c/webtransport/issues/251#issuecomment-843800902:
```js
wt.datagrams.incomingMaxAge = 300; // ms
wt.datagrams.outgoingMaxAge = 800; // ms
```
I'm also wondering if we should take a cue from recent changes to bidirectional streams, and subclass readable and writable stream here, so we can push the attributes down:
```js
wt.datagrams.readable.maxAge = 300; // ms
wt.datagrams.writable.maxAge = 800; // ms
```

Reply:
One concern is it can return null (when not set/the browser default behavior). Are you fine with the following IDL?

```
attribute double? incomingMaxAge;
attribute double? outgoingMaxAge;
```

I share [the sentiment](https://github.com/w3c/webtransport/issues/210#issuecomment-789367240) about subclassing with @ricea and I'd like to avoid that without a clear benefit. I'm not sure this is enough.

Reply:
Meeting:
- Might be better to expose the implementation-defined default than return `null`?

Reply:
Please note that the implementation-default value can be chosen for each receiveDatagram call when  [[IncomingDatagramsExpirationDuration]]. So exposing a fixed value reduces the flexisibility. Ditto for the outgoing direction.